src/components/FileRepo.js

```diff
@@ -168,13 +185,43 @@ const AggregationsWrapper = compose(injectState, withTheme)(
               },
               component: (
                 <AdvancedFacetViewModalContent
-                  {...props}
-                  {...{ statsConfig }}
-                  closeModal={effects.unsetModal}
-                  onSqonSubmit={({ sqon }) => {
-                    setSQON(sqon);
-                    effects.unsetModal();
+                  {...{
+                    ...props,
+                    closeModal: effects.unsetModal,
+                    onClear: () => {
```

Comment:
onClear, onFilterChange, onTermSelected, weren't previously in `props` ? just want to make sure we're not overwriting these handlers that may have been passed down from parent components

Reply:
double checked, these are all props that don't current exist on AFV. I put these in this PR so that when these events get merged to Arranger the file repo tracking will be all set.

Reply:
👍